homeassistant/components/device_tracker/upc_connect.py

```diff
@@ -31,12 +31,9 @@
 })
 
 
-async def async_get_scanner(hass, config):
+def get_scanner(hass, config):
```

Comment:
Why is this changed needed?

Reply:
i removed the async here because it was not needed. the return value is directly generated from the object.

Reply:
Revert this change, you don't know what you're doing. async is faster than sync methods.

Reply:
@balloob will you also explain why when you say "you don't know what you're doing"? because i don't see a single reason to make a method async which is not waiting in any case.